src/components/Toggle/index.tsx

```diff
@@ -16,9 +16,9 @@ interface ToggleProps {
 
 const Toggle: React.FC<ToggleProps> = ({
   activeColor = COLORS.waterspout,
-  inactiveColor = COLORS.lightgrey,
+  inactiveColor = COLORS.gray100,
   activeButtonColor = COLORS.anon_primary,
-  inactiveButtonColor = COLORS.lightgrey,
+  inactiveButtonColor = COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color changes seem to be consistent with the PR's purpose of updating colors across various components. However, ensure that these new color constants (`COLORS.gray100` and `COLORS.anon_primary`) are defined in your `COLORS` object and they match the desired color codes.

```diff
-  inactiveColor = COLORS.lightgrey,
+  inactiveColor = COLORS.gray100,
   activeButtonColor = COLORS.anon_primary,
-  inactiveButtonColor = COLORS.lightgrey,
+  inactiveButtonColor = COLORS.gray100,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->